### DISCUSSION TITLE
🐛: Login credentials not working as expected

### DISCUSSION THREAD

**Discussion by VJ12More** [code-a2z/jarvis#207] [Show and tell]
While logging in even with correct email id and password it is showing incorrect credentials . 
Also give an option to login through google account directly .
![Screenshot 2024-10-27 212425](https://github.com/user-attachments/assets/d9f735eb-317e-4c29-b25e-782061d1f56b)

**Comment by Avdhesh-Varshney** [comment]
Yes, we are working on it
